src/components/common/checkbox.js

```diff
@@ -93,14 +93,16 @@ export default class Checkbox extends Component {
       onBlur: this.handleBlur
     };
 
+    const {className = ''} = this.props;
+
     const labelProps = {
       ...pick(this.props, ['checked', 'disabled', 'secondary']),
       htmlFor: this.props.id
     };
 
     const LabelElement = this.props.type === 'checkbox' ? StyledCheckboxInput : StyledSwitchInput;
     return (
-      <StyledCheckbox className="kg-checkbox">
+      <StyledCheckbox className={`kg-checkbox ${className}`}>
```

Comment:
You can use `classnames` to handle the composition of classNames, to avoid blank spaces, it ignores falsy value
`className={classnames('kg-checkbox', className)}`, no need to call `const {className = ''} = this.props;`